schemas/api_schema.js

```diff
@@ -0,0 +1,269 @@
+var { makeExecutableSchema } = require("graphql-tools");
+var FormatError = require("easygraphql-format-error");
+
+// Don't return more than MAX_SESSIONS_COUNT sessions in a single request!
+const MAX_SESSIONS_COUNT = 50;
+
+const errors = new FormatError([
+  {
+    name: "COUNT_TOO_HIGH",
+    message: "The passed count value is too high",
+    statusCode: 400,
+  },
+]).errorName;
+
+function strip_null(obj) {
+  Object.keys(obj).forEach((key) => obj[key] == null && delete obj[key]);
+  return obj;
+}
+
+// TODO: remove when login is implemented
+// TODO: all instances of user_token should be swapped with some real login auth token
+const USER_AUTHENTICATED = true;
+
+function check_auth(token) {
+  if (USER_AUTHENTICATED) {
+    // TODO: switch to the real user id
+    return "1";
+  }
+
+  throw new Error(errors.UNAUTHORIZED);
+}
+
+const schema = makeExecutableSchema({
+  typeDefs: [
+    `
+scalar Date
+
+type User {
+    username: String!
+    email: String
+    display_name: String
+    is_admin: String
+    is_premium: String
+}
+
+enum Platform {
```

Comment:
a. oops
b. no, I created this enum in the beginning and didn't change it since
c. it doesn't work like that really, graphql schemas should be related to but independent from the sql one